### ISSUE TITLE
Why there is no label section in the dataset

### ISSUE THREAD

**Issue by haomingHu-dd** [yujheli/Pitt-Radar#5]
hello，Why there is no label section in the dataset，and how should I divide the training set and the test set？

**Comment by yujheli** [comment]
Hi, the task in the project is radar reconstruction, so no label is needed. The train/test is randomly selected for evaluating the super-resolution (we used 5000 samples in the paper).

**Comment by haomingHu-dd** [comment]
thank you for your reply, Good luck with your work.
